### DISCUSSION TITLE
Perform seismic tomography inversion with both refracted and reflected travel times

### DISCUSSION THREAD

**Discussion by kerim371** [gimli-org/pyGIMLi#492] [General]
Hi,

As far as I know there is commercial software that allow to calculate velocity model based on both refracted (first breaks) and reflection travel times.

Is it possible to do that with `pygimli`?

**Comment by halbmy** [comment]
Simple answer: no. Our Eikonal solver only computes first-arrivals

**Reply by kerim371** [reply]
ok, good to know anyway
